mmgen/core/evaluation/metrics.py

```diff
@@ -406,8 +407,11 @@ def feed(self, batch, mode):
                 end = min(batch_size,
                           self.num_real_need - self.num_real_feeded)
                 batch_to_feed = batch[:end, ...]
+
+            global_end = min(batch_size * ws,
+                             self.num_real_need - self.num_real_feeded)
```

Comment:
You may carefully check whether feed extra images here.

Reply:
Extra images will be dropped in `feed_op` or `summary`.